src/main/groovy/org/gradlefx/tasks/adt/AdtTask.groovy

```diff
@@ -78,15 +80,16 @@ public class AdtTask extends DefaultTask {
         adtArguments.addAll(args)
     }
 
-    def handleIfFailed(String antResultProperty, String antOutputProperty) {
+    def handleIfFailed(String antResultProperty, String antOutputProperty, String antErrorProperty) {
```

Comment:
In the cases I tested `antOutputProperty` seems to be an empty string and `antErrorProperty` holds the error text. But there could be cases when `antOutputProperty` holds something useful, and the extra empy string added to the log wont hurt anyone.